### ISSUE TITLE
[CAL-1734] Clarify what Sender ID means in Workflows

### ISSUE THREAD

**Issue by shirazdole** [calcom/cal.com#9034]
There's a portion that says "Sender ID" when setting up a workflow when the action is "Send SMS" it may exist for other actions too. 

No idea what this Sender ID means or how it is presented for the user. This should be explained with some explainer text below the field. 

![CleanShot 2023-05-22 at 10.57.51@2x.png](https://uploads.linear.app/e86bf957-d82f-465e-b205-135559f4b623/3f6e4dce-ca73-42ba-9baa-e676a51f06a5/9ad23ce7-81af-4787-9b13-81a2ed65c5bf)

![CleanShot 2023-05-22 at 11.59.52@2x.png](https://uploads.linear.app/e86bf957-d82f-465e-b205-135559f4b623/9640628d-8884-479f-8669-9ea45b14f55d/f6438de7-f9fa-4d9f-acc4-214d20a9fc17)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1734](https://linear.app/calcom/issue/CAL-1734/clarify-what-sender-id-means-in-workflows)</sub>

**Comment by CarinaWolli** [comment]
Instead of the phone number, it will show the Sender ID when you receive the SMS. Also, it is probably worth mentioning 
that some countries do not allow alphanumeric sender IDs and if that's the case we ignore the sender id, and the twilio phone number is used

Does that make it clearer? @shirazdole 
<img width="732" alt="Screenshot 2023-05-22 at 13 20 53" src="https://github.com/calcom/cal.com/assets/30310907/4f154a6b-2262-4a1c-854c-bdc2cfa848e7">

**Comment by CarinaWolli** [comment]
Exactly, the default is 'Cal' (the dot isn't allowed, that's why not Cal.com)

So it looks like that instead of showing a number: 
![IMG_2702](https://github.com/calcom/cal.com/assets/30310907/915116ae-6e8a-44ca-a7fa-58f8a7cf3903)

The US does not allow alphanumeric sender IDs, here is a list that shows if a country supports it or not: https://support.twilio.com/hc/en-us/articles/223133767-International-support-for-Alphanumeric-Sender-ID

**Comment by shirazdole** [comment]
Ohhhh yes, we should some how clarify better that this is the name that will be displayed when sending a message… I realize now that's what "Sender ID" means hahah but we need to put it in laymans terms

**Comment by Ninad3721** [comment]
@shirazdole so can the text be like :

1)Displayed value identifying who sent an SMS message, can be a phone number, name or shortcode.
2)Sender ID is the name or number that is displayed as the sender of an SMS message to the receiver.
3)A sender ID is an alphanumeric value that identifies the sender of an SMS message.
